app/services/container_dashboard_service.rb

```diff
@@ -78,4 +80,71 @@ def providers
       }
     ]
   end
+
+  def heatmaps
+    # Get latest daily rollup for each node.
+    metrics = MetricRollup.where(:resource_type => ContainerNode.name, :capture_interval_name => "daily")
+    metrics = metrics.where(:resource_id => @ems.container_nodes) if @ems
+    metrics = metrics.order(:resource_id, :timestamp => :desc)
+    metrics = metrics.select('DISTINCT ON (metric_rollups.resource_id) metric_rollups.*')
+    metrics = metrics.includes(:resource)
```

Comment:
> This also concerns me with respect to Time Profiles. See the VimPerformanceDaily class. We may need to do something different, or perhaps that class has everything that you need.

@Fryguy Im looking at ```VimPerformanceDaily``` but I dont really understand the role of TimeProfiles could you give me some code examples of how to use it?